its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SolutionKindTest.java

```diff
@@ -102,43 +103,47 @@ void testRazorCompilationNet9WithSourceGenerators() throws IOException {
 
   @Test
   void testCSharpAllFlat() throws IOException {
-    runAnalysis("CSharpAllFlat");
+    var analysisContext = AnalysisContext.forServer("CSharpAllFlat");
+    analysisContext.build.addArgument("CSharpAllFlat.sln");
```

Comment:
The previous version of "build" added `foldername + ".sln"` to each build command. This is no longer the case but needed in this case because there are `csproj` and `sln` files in this folders and msbuild wants to know which one it should build.
I think the current design is better.